test/react/rails/railtie_test.rb

```diff
@@ -0,0 +1,14 @@
+require "test_helper"
+
+class RailtieTest < ActionDispatch::IntegrationTest
+  test "reloaders are configured after initializers are loaded" do
+    begin
+      @test_file = File.expand_path("../../dummy/app/pants", File.dirname(__FILE__))
```

Comment:
This file name, does it end in `.js` or `.jsx`? I think only those files are watched due to `config.react.server_renderer_extensions`. Could that be why the test doesn't pass?